app/celery/scheduled_tasks.py

```diff
@@ -332,11 +368,14 @@ def send_scheduled_comp_and_pen_sms():
                         notification_type=SMS_TYPE,
                         personalisation={'paymentAmount': payment_amount},
                         sms_sender_id=service.get_default_sms_sender_id(),
-                        recipient_item={
-                            'id_type': IdentifierType.VA_PROFILE_ID.value,
-                            'id_value': vaprofile_id,
-                        },
+                        recipient=recipient,
+                        recipient_item=recipient_item,
                     )
+
+                    if perf_to_number is not None:
+                        current_app.logger.info(
+                            'Notification sent using Perf simulated number %s instead of vaprofile_id', perf_to_number
+                        )
```

Comment:
These lines cannot raise an exception.  Please move them out of the "try" and into the "else" starting on line 390.

This recommendation is consistent with Pep8, which prescribes to, "limit the try clause to the absolute minimum amount of code necessary."